tests/core/test_scripts.py

```diff
@@ -2657,17 +2675,19 @@ def test_do_run(self, db: DatabaseTransactionFixture):
         assert kwargs["subject"].__contains__("Inventory Report")
         attachments: dict = kwargs["attachments"]
 
-        assert len(attachments) == 1
-        key = [*attachments.keys()][0]
-        assert "biblioboard" in key
-        value = attachments[key]
-        assert len(value) > 0
-        csv_file = StringIO(value)
-        reader = csv.reader(csv_file, delimiter=",")
+        assert len(attachments) == 2
+        inventory_report_key = [x for x in attachments.keys() if "inventory" in x][0]
+        assert inventory_report_key
+        assert "test_library" in inventory_report_key
+        inventory_report_value = attachments[inventory_report_key]
+        assert len(inventory_report_value) > 0
+        inventory_report_reader = csv.reader(
```

Comment:
I think these tests would be a lot easier to read if you used a [csv.DictReader](https://docs.python.org/3/library/csv.html#csv.DictReader) here. Then you don't have to do all the manipulation with indexes and finding the first row etc.